Some iOS WhatsApp DBs are not parsed

When selecting from group members table (ZWAGROUPMEMBER), some members (ZMEMBERJID column) are null, causing a NPE in WhatsAppParser. I had this exception processing a recently generated UFDR, using the master branch. I just checked, and it also affects 4.1.6.

```text
org.apache.tika.exception.TikaException: WAExtractorException Exception
	at iped.parsers.whatsapp.WhatsAppParser.parseWhatsappMessages(WhatsAppParser.java:394)
	at iped.parsers.whatsapp.WhatsAppParser.parse(WhatsAppParser.java:265)
	at org.apache.tika.parser.CompositeParser.parse(CompositeParser.java:298)
	at iped.parsers.standard.StandardParser.parse(StandardParser.java:245)
	at iped.engine.io.ParsingReader$BackgroundParsing.run(ParsingReader.java:247)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException
	at iped.parsers.whatsapp.Extractor.setGroupMembers(Extractor.java:65)
	at iped.parsers.whatsapp.ExtractorIOS.extractChatList(ExtractorIOS.java:257)
	at iped.parsers.whatsapp.Extractor.getChatList(Extractor.java:34)
	at iped.parsers.whatsapp.WhatsAppParser.parseWhatsappMessages(WhatsAppParser.java:386)
```

--- a/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/Extractor.java
+++ b/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/Extractor.java
@@ -62,11 +62,10 @@ public abstract class Extractor {
 
                 while (rs.next()) {
                     String memberId = rs.getString("member");
-                    if (!memberId.trim().isEmpty()) {
+                    if (memberId != null && !memberId.trim().isEmpty()) {
                         c.getGroupMembers().add(contacts.getContact(memberId));
                     }
                 }
-
             }
         }
     }

